doc/contributing.rst

```diff
@@ -491,8 +552,7 @@ Writing tests
 All tests should go into the ``tests`` subdirectory of the specific package.
 This folder contains many current examples of tests, and we suggest looking to these for
 inspiration.  If your test requires working with files or
-network connectivity, there is more information on the `testing page
-<https://github.com/pydata/xarray/wiki/Testing>`_ of the wiki.
+network connectivity.
```

Comment:
Now this sentence is incomplete. Why did you remove the rest?

Reply:
https://github.com/pydata/xarray/wiki/Testing -> this is a broken link. And I could not find any wiki page on xarray.
So I deleted the link, I did not remove the entire sentence.

I'll remove the entire sentence.